beast/fitting/pdf1d.py

```diff
@@ -86,6 +86,22 @@ def __init__(self, gridvals, nbins, logspacing=False, minval=None, maxval=None):
             self.pdf_bin_indxs = pdf_bin_indxs
 
     def gen1d(self, gindxs, weights):
+        """Determines the probabilities for each bin in weights.
```

Comment:
What does it means "in weights"?  Do you mean "using the grid+prior weights and the likelihoods"?

Reply:
And maybe "Calculate" instead of "Determine"?

Reply:
I was trying to clarify that the "weights" here are the values for the combined (log likelihood, grid weights, and priors). Calculate works for me.

Reply:
Also I thought the bins were determined by the grid weights. Hopefully that's correct and the correction makes that more clear.